src/app/[affiliateSlug]/layout.tsx

```diff
@@ -1,93 +1,153 @@
-import config from "@/config/env_config";
-import type { Metadata } from "next";
-import { Inter } from "next/font/google";
-import { headers } from "next/headers";
-import Script from "next/script";
-import "../globals.css";
+import getCompany from "@/api/getCompany"
+import config from "@/config/env_config"
+import {
+	css,
+	fresponsive,
+	GlobalStyles,
+	styled,
+	unresponsive,
+} from "library/styled"
+import type { Metadata } from "next"
+import { headers } from "next/headers"
+import Script from "next/script"
+import Footer from "@/app/components/Footer"
+import Header from "@/app/components/Header"
+import GlobalProviders from "@/app/components/Providers"
 
-const inter = Inter({ subsets: ["latin"] });
+import colors from "styles/colors"
+import "../globals.css"
 
 declare global {
-  interface Window {
-    fcs: any;
-    fluidChatSettings: any;
-    fluidSettings: any;
-    Spreedly: any;
-    showCartCount: Function;
-  }
+	interface Window {
+		fcs: any
+		fluidChatSettings: any
+		fluidSettings: any
+		Spreedly: any
+		showCartCount: () => any
+	}
 }
 
 type PageProps = Readonly<{
-  children: React.ReactNode;
-  params: Record<string, string>;
-}>;
+	children: React.ReactNode
+	params: Promise<Record<string, string>>
+}>
 
 export default async function RootLayout({ children, params }: PageProps) {
-  const { affiliateSlug } = params;
-  const company = await getCompany();
+	const resolvedParams = await params
+	const { affiliateSlug } = resolvedParams
 
-  return (
-    <html lang="en">
-      <head>
-        <Script id="fluid-widget-boot" strategy="beforeInteractive">
-          {`
+	return (
+		<html lang="en">
+			<head>
+				<Script id="fluid-widget-boot" strategy="beforeInteractive">
+					{`
           window.fcs = {api_url_host: '${config.apiHost}', affiliate: { credit: '${affiliateSlug}' }};
           (function(){ var f_ws = document.createElement('script'); f_ws.async = true; f_ws.src = '${config.widgetHost}'; x = document.getElementsByTagName('script')[0]; x.parentNode.insertBefore(f_ws,x); })();
         `}
-        </Script>
-        <meta charSet="utf-8" />
-        <meta
-          name="viewport"
-          content="width=device-width, initial-scale=1.0"
-        ></meta>
-        <link
-          rel="icon"
-          href={company.logo_url || "https://cdn.fluid.app/favicon-16x16.png"}
-        />
-      </head>
-      <body className={`${inter.className} h-screen`}>
-        <Navbar params={params} company={company} />
-        {children}
-        <Footer params={params} company={company} />
-      </body>
-    </html>
-  );
+				</Script>
+				<meta charSet="utf-8" />
+				<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+			</head>
+			<PageRoot suppressHydrationWarning>
+				<GlobalProviders>
+					<GlobalStyles>{globalCss}</GlobalStyles>
+					<Header />
+					{children}
+					<Footer />
+				</GlobalProviders>
+			</PageRoot>
+		</html>
+	)
 }
 
+const PageRoot = styled(
+	"body",
```

Comment:
PageRoot wasn't intended to be the body, it’s only meant to wrap `header`, `main`, and `footer`